src/credentials/FabricTable.h

```diff
@@ -644,6 +655,33 @@ class DLL_EXPORT FabricTable
      */
     CHIP_ERROR FetchNOCCert(FabricIndex fabricIndex, MutableByteSpan & outCert) const;
 
+    /**
+     * @brief Recover the VIDVerificationStatement for a fabric, if any exists.
+     *
+     * This includes returning a pending one if one exists.
+     *
+     * @param fabricIndex - Fabric index for which to find a VIDVerificationStatement
+     * @param outVIDVerificationStatement - Buffer to receive the VIDVerificationstatement. Must be large enough for the maximum
+     * size.
+     *
+     * @retval CHIP_NO_ERROR on success, **including if missing**.
```

Comment:
can we say something on the is_missing case? Would the out size be 0 in that case?

Reply:
Will do in a follow-up.